.eslintrc.json

```diff
@@ -1,3 +1,78 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": [
+    "next/core-web-vitals",
+    "prettier"
+  ],
+  "rules": {
+    "semi": "error",
```

Comment:
This (and some other rules below) should be handled automatically by Prettier, but we can keep it as a protection.